packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -68,6 +90,92 @@ export const certificateTransactionsWithEpochs = (
     )
   );
 
+export const createDelegationPortfolioTracker = ({
+  rewardAccounts$,
+  knownAddresses$,
+  utxoTracker
+}: DelegationPortfolioTrackerProps): Observable<Cip17Pool[]> => {
+  const balanceAllUtxosTracker = createBalanceByAddressTracker(utxoTracker);
+
+  const delegatedAccounts$ = rewardAccounts$.pipe(
+    map((rewardsAccounts) =>
+      rewardsAccounts.filter(
+        (account) => account.keyStatus === Cardano.StakeKeyStatus.Registered && account.delegatee?.nextNextEpoch
```

Comment:
REVIEW: as far as I could understand, nextNextEpoch will always reflect the latest delegation changes, even if they will be applied 2 epochs from now.
I chose this property as the source of truth when building the cip17 object

Reply:
The most flexible solution seems to be #3.

Reply:
Leave it as is for now